utils/playwright_utils.py

```diff
@@ -0,0 +1,13 @@
+
+class PlaywrightUtils:
+    def __init__(self, page):
+        self.page = page
+
+    def wait_and_get_element(self, locator):
```

Comment:
two notes here:
- locators are instances of Locator class created out of selectors. I think it's more appropriate to call this a selector
- I don't think there's an auto-wait happening here. For an explicit wait we could take the selector and use this approach: https://playwright.dev/docs/api/class-page/#page-wait-for-selector